src/main/java/roomescape/repository/ReservationRepository.java

```diff
@@ -3,19 +3,32 @@
 
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.jdbc.core.JdbcTemplate;
+import org.springframework.jdbc.core.PreparedStatementCreator;
 import org.springframework.jdbc.core.RowMapper;
+import org.springframework.jdbc.support.GeneratedKeyHolder;
+import org.springframework.jdbc.support.KeyHolder;
 import org.springframework.stereotype.Repository;
 import roomescape.model.Reservation;
 import roomescape.model.ReservationTime;
 
-import java.sql.ResultSet;
-import java.sql.SQLException;
+import java.sql.*;
 import java.util.List;
 
 @Repository
 public class ReservationRepository {
     private final JdbcTemplate jdbcTemplate;
 
+    private Reservation mapReservation(ResultSet rs, int rowNum) throws SQLException {
+        Long reservationId = rs.getLong("reservation_id");
+        String reservationName = rs.getString("reservation_name");
+        String reservationDate = rs.getString("reservation_date");
+        Long timeId = rs.getLong("time_id");
+        String startAt = rs.getString("start_at");
+
+        ReservationTime time = new ReservationTime(timeId, startAt);
+        return new Reservation(reservationId, reservationName, reservationDate, time);
+    }
```

Comment:
```suggestion
    private final RowMapper<Reservation> rowMapper = (resultSet, rowNum) -> {
        Long reservationId = resultSet.getLong("reservation_id");
        String reservationName = resultSet.getString("reservation_name");
        String reservationDate = resultSet.getString("reservation_date");
        Long timeId = resultSet.getLong("time_id");
        String startAt = resultSet.getString("start_at");

        ReservationTime time = new ReservationTime(timeId, startAt);
        return new Reservation(reservationId, reservationName, reservationDate, time);
    };
```
이런 식으로 람다식을 선언해서 사용하면 조금 더 간결하게 표현할 수 있겠네요 :)